patternlab/styleguide/source/assets/js/modules/mainNav.js

```diff
@@ -1,232 +1,301 @@
-export default (function (window, document, $, undefined) {
+const body = document.querySelector("body");
+let width = body.clientWidth;
+const menuButton = document.querySelector(".js-header-menu-button");
+let menuButtonText = menuButton.querySelector('.ma__header__menu-text');

```

Comment:
We use a unique prefix for selectors for JS, 'js-' to separate one for css 'ma__' like `<div class="ma__header__search js-header-search-menu">`. 

If the existing markup doesn't have one for JS, feel free to add one.  
This way, we can manage selectors independently between JS and styles without risking breaking either styles or JS as we refactor templates, styles or JS.